[EN COURS] fix issue 1666 : modales

cyrilphoenix71 opened the pull request:
fix #1666
Add blur when modale

To test :
- check if height of modals is ok now with height résolution change 
- check all the update page (modale log, update counters etc)
- check all the backup page (modal log, backup, restore)
- test modale icon selector
- check the scenarios' log modals

NE PAS MERGER

vinceg77 commented:
I would add to test : 
* check all the scenario log modals

iWils reviewed (approved):
PR testé et validé.

- L'effet blur est bien présent sur les fonds lors de l'affichage des modales

- Les modales ce redimensionne bien en largueur et en hauteur.

cyrilphoenix71 committed:
fix 1666


fix #1666


fix sonar


fix


Modification ajax et manager

optimize notif update


phase 2


fix


increment update


fix
